extension_distribution.md

```diff
@@ -225,6 +241,21 @@ type TiltfileStateTerminated {
 }
 ```
 
+### Tiltfile API
+
+To start, users will install extensions from the main Tiltfile. The syntax will look like this:
+
+```
+extension_repo(name='default', url='https://github.com/tilt-dev/tilt-extensions')
+extension(name='cancel_button', repo_name='default', repo_path='cancel_button')
+```
+
+We have deliberately designed this API so that it can be auto-generated from the above models.
```

Comment:
A `tilt_yaml()` type function might also be handy, especially for the interim before auto-generation exists so that we can quickly prototype using new API types in Tiltfile w/o shelling out to `echo $YAML | tilt apply -f -`

Reply:
yaaaa